src/WebJobs.Script/Config/HostTelemetryInitializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    internal class HostTelemetryInitializer : ITelemetryInitializer
+    {
+        private const string HostInstanceIDKey = ScriptConstants.LogPropertyHostInstanceIdKey;
```

Comment:
No need to create a const that refers to another const -- just use the `ScriptConstants` one directly.

Reply:
Sounds good. Wasn't sure if the const I am using was meant for something else and wanted an easy way to switch. But, I think you are right, I will update.